EasyPost.Net/WebhookList.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace EasyPost
 {
-    public class WebhookList : Resource
+    internal class WebhookList : Resource
```

Comment:
Marking it internal makes it inaccessible to anything outside of the EasyPost client library. Currently only used internally anyway (or at least, should be). Confusing for the end user if they need to use `WebhookList` (incorrect) or `WebhookCollection` (correct). This will remove it from any code-completion suggestions.